easybuild/easyconfigs/s/statsmodels/statsmodels-0.9.0-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'statsmodels'
+version = '0.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.statsmodels.org'
+description = """Statsmodels is a Python module that provides classes and functions
+for the estimation of many different statistical models, as well as for conducting 
+statistical tests, and statistical data exploration."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+
+exts_list = [
+    ('patsy', '0.5.1', {
+        'source_urls': [PYPI_SOURCE],
```

Comment:
@jose-d afaik, templates don't work (yet) on extension lists, you need to explicitly add

 `'source_urls': ['https://pypi.python.org/packages/source/p/patsy'],`

Reply:
I see and understood the reason why it ~worked~ hasn't failed locally on my machines.. First (just for testing) I used harcoded URL so tar.gz was already in my download folder..

But I'm not sure why it says **All checks have passed** here - with technically non-existing URL(?) I thought the checks are stateless(?)

Reply:
done.

Reply:
@jose-d the travis tests don't actually try to download and install anything, for that we have the test reports